app/src/main/java/com/courseproject/tindar/entities/Account.java

```diff
@@ -0,0 +1,311 @@
+     * @param email the email associated with this account
+     * @param password the password used to log in to this account
+     * @param display the username chosen to represent this account
+     * @param first the account owner's first name
+     * @param last the account owner's last name
+     */
+    Account(String id, String email, String password,
+                   String display, String first, String last) {
+        this.isActiveStatus = true;
+        this.id = id;
+        this.email = email;
+        this.password = password;
+        this.displayName = display;
+        this.firstName = first;
+        this.lastName = last;
+        this.aboutMe = "";
+        this.profilePictureLink = "";
+        this.matchList = new ArrayList<>();
+        this.likeList = new ArrayList<>();
+        this.blockList = new ArrayList<>();
+        this.recentlyViewed = new ArrayList<>();
+        this.birthdate = null;
+        this.location = null;
+        this.gender = null;
+        this.filters = null;
+    }
+    /** Returns the status of an account in the form of a String.
+     *
+     * @return the status of the account
+     */
+    String getAccountStatus() {
+        if (this.isActiveStatus) {
+            return "Active";
+        }
+        else {
+            return "Suspended";
+        }
+    }
+    /** Flips the account status from Active to Suspendedor vice versa.
+     *
+     */
+    void setAccountStatus() {
+        this.isActiveStatus = !this.isActiveStatus;
+    }
+    /** Returns the id of an account in the form of a String.
+     *
+     * @return the user id associated with the account
+     */
+    String getId() {
+        return this.id;
+    }
+    /** Returns the email of an account in the form of a String.
+     *
+     * @return the email associated with the account
+     */
+    String getEmail() {
+        return this.email;
+    }
+    /** Changes the email associated with the account.
+     *
+     * @param email the new email associated with the account
+     */
+    void setEmail(String email) {
+        this.email = email;
+    }
+    /** Returns the password of an account in the form of a String.
+     *
+     * @return the password associated with the account
+     */
+    String getPassword() {
+        return this.password;
+    }
+    /** Changes the password associated with the account.
+     *
+     * @param password the new password associated with the account
+     */
+    void setPassword(String password) {
+        this.password = password;
+    }
+    /** Returns the display name of an account in the form of a String.
+     *
+     * @return the display name associated with the account
+     */
+    String getDisplayName() {
+        return this.displayName;
+    }
+    /** Changes the display name associated with the account.
+     *
+     * @param display the new display name associated with the account
+     */
+    void setDisplayName(String display) {
```

Comment:
Can we have parameter name as displayName instead of display to make it clearer?